src/main/java/com/untamedears/JukeAlert/storage/JukeAlertLogger.java

```diff
@@ -1632,11 +1632,19 @@ public boolean updateMutedGroups(UUID uuid, String group2Mute) {
 			return null;
 		} else {
 			// Add the first value
-			ignoringUsers.add(set.getString(1));
+			String[] mutedGroups = set.getString(2).split("\\s+");
+			List<String> mutedGroupsSet = new ArrayList<String>(Arrays.asList(mutedGroups));
```

Comment:
I probably wasn't clear.

Arrays.asList(mutedGroups) --- all by itself, don't wrap it with anything else.